### DISCUSSION TITLE
Arcusin forstack 8.12

### DISCUSSION THREAD

**Discussion by le-Vinyl** [Courseplay/Courseplay_FS22#2746] [Q&A]
Hallo. 
Ist es möglich mit dem Arcusin und Courseplay die 3 Ballen zu sammeln und automatisch auf der bahn zu entladen? Das alle paar Meter ein stappel steht.

Liebe Grüße

**Comment by Tensuko** [comment]
Ja, mit dem Gerät kann CP arbeiten.
Ob das mit Kurs geht habe ich nie probiert, aber im Modus ohne Kurs funktioniert es.

**Reply by le-Vinyl** [reply]
Ja das funktioniert schon mit und ohne Modus allerdings nur das sammeln. Sobald er voll ist, kommt die Meldung das er entladen werden muss. Schön wäre es wenn er automatisch entlädt und weiter arbeitet. Zumin im Modus

**Reply by Tensuko** [reply]
Auch ohne Kurs?
Es sollte an sich funktionieren.

**Reply by le-Vinyl** [reply]
Hallo. 
Sorry für die späte Antwort aber ich brauchte erst wieder Stroh zum Pressen ;)
Also mit dem course "ballen sammeln" fährt er quer über das Feld und sammelt alle 3 Ballen auf und bleibt im Anschluß mit der Meldung "muss entladen werden" stehen.
Lade ich den Kurs der Strohpresse nimmt er auch alle 3 auf aber bleibt stehen ohne jede Meldung.
Leider entlädt er sich auf keiner Art von allein.
Liebe Grüße

**Reply by Tensuko** [reply]
Ok muss ich mal gucken.

**Reply by Tensuko** [reply]
Ja der geht nicht, aber ich hab eben geguckt, der Trigger zum Sammeln der Ballen ist auch echt bescheiden.
Ist eher low priority, vlt findet @schwiti6190 eine Lösung.
![grafik](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/7d52bfcc-5e30-4828-a9ef-1570ac1b3c53)

**Reply by le-Vinyl** [reply]
Ok. Danke dir
